apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-title/index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Render site title block.
+ *
+ * @package full-site-editing
+ */
+/**
+ * Renders the site title and allows for editing in the full site editor.
+ *
+ * @param array  $attributes Block attributes.
+ * @param string $content    Block content.
+ * @return string
+ */
+function render_site_title_block( $attributes, $content ) {
+	ob_start();
+	?>
+	<h1 class="site-title wp-block-a8c-site-title">
```

Comment:
Note that a user can define a custom class for this block:

<img width="1167" alt="Screen Shot 2019-06-17 at 15 00 08" src="https://user-images.githubusercontent.com/1233880/59581771-e50f4c00-9110-11e9-9a30-77edeab64924.png">

We should check if there is any value on the `className` attribute and use it on the HTML output:

```suggestion
	$class = 'site-title wp-block-a8c-site-title';
	if ( isset( $attributes['className'] ) ) {
		$class .= ' ' . $attributes['className'];
	}
	?>
	<h1 class="<?php echo esc_attr( $class ); ?>">
```

Reply:
Thanks so much! Learning all the things about block development 🌈